File: spark-data-visualization/src/main/java/cn/edu/nju/domain/CourseClickCount.java
package cn.edu.nju.domain;

import org.springframework.stereotype.Component;

/**
 * Created by Thpffcj on 2018/1/15.
 * 实战课程访问数量实体类
 */
@Component
public class CourseClickCount {

    private String name;
    private long value;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getValue() {
        return value;
    }

    public void setValue(long value) {
        this.value = value;
    }
}

File: spark-train/src/main/scala/cn/edu/nju/spark/project/domain/CourseClickCount.scala
package cn.edu.nju.spark.project.domain

/**
  * Created by Thpffcj on 2018/1/15.
  * 实战课程点击数实体类
  * @param day_course  对应的就是HBase中的rowkey，20171111_1
  * @param click_count 对应的20171111_1的访问总数
  */
case class CourseClickCount(day_course:String, click_count:Long)
